integration/test/test_multi_app.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#  Copyright (c) 2015 - 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+cat > temp_config.json << "EOF"
+{
+    "loop-count": 2,
+    "region": ["stream", "dgemm"],
+    "big-o": [1.0, 1.0]
+}
+EOF
+
+source ${GEOPM_SOURCE}/integration/config/run_env.sh
+
+TEST_NAME=test_multi_app
+export GEOPM_REPORT=${TEST_NAME}_report.yaml
+export GEOPM_TRACE=${TEST_NAME}_trace.csv
+export GEOPM_TRACE_PROFILE=${TEST_NAME}_trace_profile.csv
+export GEOPM_PROFILE=${TEST_NAME}
+#export GEOPM_REPORT_SIGNALS=CPU_TIMESTAMP_COUNTER@package
+export GEOPM_REPORT_SIGNALS=TIME@package
+export GEOPM_NUM_PROC=2
+
+geopmctl &
+sleep 2
```

Comment:
geombench has a sleep 5 in it.  Is this still needed?  If yes, please add a comment explaining why.  I'm guessing this is just a problem for stress-ng?

Reply:
Remove the sleep and re-test.